src/application/services/useNoteSettings.ts

```diff
@@ -79,10 +92,21 @@ export default function (): UseNoteSettingsComposableState {
     }
   };
 
+  const joinNote = async (hash: string): Promise<TeamMember> => {
```

Comment:
use named function instead of anonymous

```suggestion
  async function joinNote(hash: string): Promise<TeamMember> {
```

Reply:
jsdoc is missed